backend/routes/forms/submit.py

```diff
@@ -133,11 +133,65 @@ async def post(self, request: Request) -> JSONResponse:
                 response_obj.dict(by_alias=True)
             )
 
+            send_webhook = None
+            if FormFeatures.WEBHOOK_ENABLED.value in form.features:
+                send_webhook = BackgroundTask(
+                    self.send_submission_webhook,
+                    form=form,
+                    response=response_obj
+                )
+
             return JSONResponse({
-                "form": form.dict(),
+                "form": form.dict(admin=False),
                 "response": response_obj.dict()
-            })
+            }, background=send_webhook)
+
         else:
             return JSONResponse({
                 "error": "Open form not found"
             }, status_code=404)
+
+    @staticmethod
+    async def send_submission_webhook(form: Form, response: FormResponse) -> None:
+        """Helper to send a submission message to a discord webhook."""
+        # Stop if webhook is not available
+        if form.meta.webhook is None:
+            raise ValueError("Got empty webhook.")
+
+        user = response.user
+        username = f"{user.username}#{user.discriminator}" if user else None
+        user_mention = f"<@{user.id}>" if user else f"{username or 'User'}"
+
+        # Build Embed
+        embed = {
+            "title": "New Form Response",
+            "description": f"{user_mention} submitted a response.",
```

Comment:
Maybe we should include the form name here?
```suggestion
            "description": f"{user_mention} submitted a response to `{form.name}`.",
```